app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -107,38 +152,41 @@ class RefereeProfile extends Component {
       )
     }
 
+    if (edit) {
+      return <RefereeProfileForm defaultValues={referee} onSubmit={this.save} />
+    }
+
     return (
-      <div>
-        <h1>
-          {`${firstName} ${lastName}`}
-        </h1>
+      <Fragment>
+        <Header as="h1">
+          {`${referee.firstName} ${referee.lastName}`}
+          {referee.isEditable && ' (Wow, that’s you!)'}
```

Comment:
Haha nice, we'll probably have to put in some better user facing text.

Reply:
Looks like there is a reason why I’m a software engineer and not a UX designer. 😄